alertmanager/alertmanager.yaml

```diff
@@ -0,0 +1,33 @@
+global:
+  resolve_timeout: 5m
+  slack_api_url: ''
+route:
+  group_by: ['alertname', 'cluster', 'service', 'deployment', 'namespace']
+  group_wait: 30s
+  group_interval: 5m
+  repeat_interval: 1h
+  receiver: default-receiver
+  routes:
+  - match:
+      alertname: DeadMansSwitch
+    receiver: 'null'
+receivers:
+- name: 'default-receiver'
+  slack_configs:
+  - channel: '#alerts'
+    title: >-
+        [{{ .Status | toUpper }}{{ if eq .Status "firing" }}:{{ .Alerts.Firing | len }}{{ end }}]
+        {{ range .Alerts }}
+        {{ .Annotations.summary }}
+        {{ end }}
+    text: >-
+        {{ range .Alerts }}
+           *Alert:* {{ .Annotations.summary }} - `{{ .Labels.severity }}`
+          *Description:* {{ .Annotations.description }}
```

Comment:
It looks like Line 25 has an extra space, as it does in the [article you and I both copy pasted it from](https://harthoover.com/pretty-alertmanager-alerts-in-slack/). 😉